pom.xml

```diff
@@ -25,7 +25,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>1.5.10.RELEASE</version>
+        <version>2.1.0.RELEASE</version>
```

Comment:
why are we upgrading to `2.1.0` and not to a newer one?

Reply:
This is the minimum version needed for the MongoDB 4.0 driver. I don't want to leap up far ahead in the version so that we keep changes to the minimum.